ramler-java/src/main/java/org/ops4j/ramler/generator/Configuration.java

```diff
@@ -222,4 +224,23 @@ public boolean isJacksonPropertyName() {
     public void setJacksonPropertyName(boolean jacksonPropertyName) {
         this.jacksonPropertyName = jacksonPropertyName;
     }
+
+    /**
+     * Whether fluent setters shall be generated
+     */
+    public boolean isFluentSetters() {
```

Comment:
I'd prefer separate properties to enable fluent setters and to define the prefix.

This would support empty prefixes, for fluent setters, which I think is a frequent naming convention, e.g.

```
new Person().firstName("Donald").lastName("Duck")
```

Maybe this is covered by `<fluentSetterPrefix/>`, I haven't tried. Separate properties would be more user-friendly.